networking.md

```diff
@@ -0,0 +1,88 @@
+# Filecoin Network
+
+The filecoin network is built using libp2p building blocks, as transports and protocols, as well as some additional filecoin specific protocols as outlined in [Network Protocols](./network-protocols.md).
+
+## Required Protocols
+
+Every full node must support the following libp2p protocols:
+
+ - [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) ([for data announcements](./data-propagation.md))
+ - [bitswap] ([for data exchange](./data-propagation.md))
```

Comment:
Missing link to bitswap

Reply:
`[for data exchange](./data-propagation.md)` links generally don't seem to since the hugo setup anymore. @dignifiedquire any idea why or how to do it instead (without breaking regular markdown links?)